includes/alerts.inc.php

```diff
@@ -459,6 +459,7 @@ function RunJail($code, $obj)
  */
 function DescribeAlert($alert)
 {
+    global $alerts_id;
```

Comment:
Don't use a global, they are too unpredictable.  Load the id from the database instead.

Reply:
K. At this point in the code, I can't get that as I don't see a way to index back to alerts.id.  I can get that in RunAlerts() so I need to stuff that somewhere to use in DescribeAlert().  Would there be a better way to do this?

Reply:
Why can't you store the id in the database? (adding a new field)

Also, looking below if you have multiple alerts, they will all use the alert_id of the last alert...